service/storage_service.hh

```diff
@@ -527,6 +527,7 @@ private:
     sharded<service::tablet_allocator>& _tablet_allocator;
     sharded<cdc::generation_service>& _cdc_gens;
     sharded<db::view::view_builder>& _view_builder;
+    std::atomic<bool> _isolated = false;
```

Comment:
Does this still need to be atomic, now that it is a regular member of `storage_service`?

Reply:
No, it really does not.

Reply:
Changed